app/views/people/_iban_form.html.haml

```diff
@@ -10,8 +10,8 @@
     ), je hoort het bedrag voor 17 augustus te ontvangen.
   %br
   .input-field
-    = f.label :iban
     = f.text_field :iban, placeholder: 'NL12 RABO 0123 4123 41', autofocus: true, required: true, pattern: "^[a-zA-Z]{2}[0-9]{2}[ ]?[a-zA-Z0-9]{4}[ ]?[0-9 ]{7}([a-zA-Z0-9 ]?){0,16}$", title: "Vul het rekeningnummer in IBAN formaat in."
+    = f.label :iban
   = render partial: 'name_fields', locals: {f: f} 
   .actions
-    = f.submit "Opslaan", class: "btn waves-effect waves-light"
+    = f.button 'Opslaan', class: 'btn waves-effect waves-light', data: { disable_with: 'Bezig...' }
```

Comment:
hier een partial van maken? Zodat we die `disable_with` niet meer vergeten?

Reply:
Fixed.